test/crit/Makefile

```
@@ -0,0 +1,19 @@
+CC ?= gcc
+
+all: test clean
+
+test: gen-imgs
+	@echo "Running E2E test"
+	@./crit-test.sh
+
+gen-imgs: loop
+	$(eval PID := $(shell ./loop))
+	criu dump -v4 -o dump.log -D ./ -t $(PID)
```

It might be good to use variable for the path to the `criu` binary: `CRIU ?= criu`
This conditional assignment would enable running the test with locally compiled criu instead.
```suggestion
	$(CRIU) dump -v4 -o dump.log -D ./ -t $(PID)
```